catalog/opensearch/catalog-opensearch-source/src/test/java/ddf/catalog/source/opensearch/TestOpenSearchSource.java

```diff
@@ -13,10 +13,9 @@
  */
 package ddf.catalog.source.opensearch;
```

Comment:
✏️  Coverage could be higher on the OpenSearchFilterVisitor methods for dealing with `And`, `Or`, `Not`, etc